DelHel/CDelHel.cpp

```diff
@@ -209,6 +212,31 @@ bool CDelHel::OnCompileCommand(const char* sCommandLine)
 			this->SaveSettings();
 			this->CheckLoadedPlugins();
 
+			return true;
+		}
+		else if (args[1] == "config")

```

Comment:
I don't have a strong preference, but is `config` a bit generic maybe (since it could also relate to configuration of the plugin itself, not necessarily the Runway Config)? Maybe `rwycfg` or something?
If you think that'd be unreadable/too difficult to remember, I'm fine with `config` as well though.